glitter/blocks/carousel/templates/glitter/blocks/carouselblock_content.html

```diff
@@ -9,7 +9,7 @@
                 {% for image in carousel_images %}
                     <li data-target="#carousel-{{ content_block.id }}" data-slide-to="{{ forloop.counter0 }}" {% if forloop.first %}class="active"{% endif %}></li>
                     <h2>{{ image.title }}</h2>
-                    <p>{{ image.subtitle }}</p>
+                    <p>{{ image.subtitle | linebreaks }}</p>
```

Comment:
Yes.

If you use `|linebreaks`, you'll end up with:

```
<p><p>Text</p></p>
```

So either switch the tag, or remove the `<p>` tags.